draft-ietf-quic-transport.md

```diff
@@ -697,6 +697,16 @@ the cryptographic handshake provides QUIC with:
   client can receive packets that are addressed with the transport address that
   is claimed by the client (see {{source-address-token}})
 
+The initial cryptographic handshake message needs to be sent in a single packet.
+This avoids having to reassemble a message from multiple packets. This would
+require that servers maintain state prior to establishing a connection, exposing
+servers to a denial of service risk.
```

Comment:
Phrasing unclear. How about "This avoids having to reassemble a message from multiple packets since doing so would require that a server maintain state prior to establishing a connection, exposing the server to a denial of service risk."